api/app/routes/user.routes.ts

```diff
@@ -93,15 +107,22 @@ export function userRoutes(app: Express) {
    * @swagger
    * /api/users/public:
```

Comment:
I can't see anywhere in the frontend we're using it. But If I remember correctly, I think we had this endpoint so you can fetch a user profile but it'll only return non-privacy invasive data. I think it was a request from Jonas. Can't vividly remember now.
But I can repurpose it so the username is passed as query params instead of in the body so that it works with swagger. We should have body for get requests

Reply:
I thought that is not a standard practice to send a request body in a GET request. If is something that is being used then I'm okay with it.

Reply:
Yes. And as you can see from my last commit, I already modified this https://github.com/bitcointranscripts/transcription-review-backend/pull/292/commits/b963659860e16f39155d6b343791480546ce545c

Reply:
Great, this is working now!

nit: Additionally, I noticed that this endpoint also returns a field called `latestReviews` that includes the last 5 reviews of the user. It is not mentioned in the swagger description. I'm not sure what's the utility of this information, but if we are going to keep it, let's document it.
Also, this endpoint returns the id of the user if the requester `isAdmin` but the id of the user is already presented in the individual reviews as part of `latestReviews`.